### PR TITLE
Refactor icons to be more accessible

### PR DESCRIPTION
**Pull request by trentmarinoIX**
This PR is related to ticket 382 in gitlab

### CODE REVIEW

File: icons/askizzy-icon-using-violence.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg class='${this.classNames}' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><g><rect width="64" height="64" fill="none"></rect><path d="M60.07,27.81a4.36,4.36,0,0,0-4.28-4.41H52.84a4.42,4.42,0,0,0,0-8.83h-18L41,12.8,37.26,9.16c-1.19-1.2-2.42-2-3.94-1.94a6.18,6.18,0,0,0-4.14,1.42l-5.66,4c-1.61,1.61-3.06,3-4.25,4.48h-1a14.35,14.35,0,0,0,0,28.7h1.6a13.22,13.22,0,0,0,9.52,4.06H42.29l6.14,0A4.42,4.42,0,0,0,49,41h3.89a4.42,4.42,0,0,0,.71-8.76h2.1l-.26-.07h.39a4.36,4.36,0,0,0,4.28-4.41" fill="#ffcc80"></path><path d="M45.43,49.85l-3.14,0H33l-3,4.26,3.82,2.67,4.41-6.3,6.29,4.4,2.67-3.82-1.73-1.21" fill="#f6695e"></path><path d="M41.42,37.76,37.05,44,30.8,39.62l-2.67,3.82,6.26,4.37-1.43,2h9.33l3.14,0-4.56-3.19,4.36-6.24-3.82-2.67" fill="#f65e45"></path></g></svg>
```
Comment (ghost) [inline icons/askizzy-icon-using-violence.svg]: You can't use `this.classNames` in this repo.

File: icons/askizzy-icon-OnlineSecurity.svg [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg class='${this.classNames}' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><title>Online security</title><g><rect width="64" height="64" fill="none"></rect><rect x="7.5" y="10.86" width="48.99" height="36.74" fill="#d4cdd1"></rect><polygon points="56.5 47.6 7.5 47.6 1.67 47.6 1.67 53.14 62.33 53.14 62.33 47.6 56.5 47.6" fill="#d4cdd1"></polygon><rect x="11.11" y="14.45" width="41.78" height="29.55" fill="#37474f"></rect><path d="M36,26.44a4.93,4.93,0,0,0-1-3.12A3.74,3.74,0,0,0,32,22a3.78,3.78,0,0,0-3,1.36,4.88,4.88,0,0,0-1,3.1v1.18H26.35v7.88H37.65V27.62H36Zm-1.69,1.18H29.74V26.44a3.27,3.27,0,0,1,.58-2A2,2,0,0,1,32,23.67a1.94,1.94,0,0,1,1.68.71,3.3,3.3,0,0,1,.58,2.06Z" fill="#d4cdd1"></path></g></svg>
```
Comment (ghost) [inline icons/askizzy-icon-OnlineSecurity.svg]: You can't use `this.classNames` in this repo.

**Review by ghost — CHANGES_REQUESTED** [review]

File: icons/askizzy-icon-using-violence.svg (hunk 1 above)
Comment (trentmarinoIX) [reply icons/askizzy-icon-using-violence.svg]: Yeah I know, It was left over when I copied the svgs over this one got missed

File: icons/askizzy-icon-OnlineSecurity.svg (hunk 2 above)
Comment (trentmarinoIX) [reply icons/askizzy-icon-OnlineSecurity.svg]: Yeah I know, It was left over when I copied the svgs over this one got missed

**Review by ghost — APPROVED** [review]

**Pull request merged**